File: messages/src/main/java/com/thundashop/zauiactivity/dto/BookingReserveRequest.java
package com.thundashop.zauiactivity.dto;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
public class BookingReserveRequest {
    private Integer productId;
    private String optionId;
    private String availabilityId;
    private String notes;
    private List<UnitItemReserveRequest> unitItems = null;
}

File: com.getshop.client/events/zauiactivity/dto/BookingReserveRequest.php
<?php
class zauiactivity_dto_BookingReserveRequest {
	/** @var String */
	public $productId;

	/** @var String */
	public $optionId;

	/** @var String */
	public $availabilityId;

	/** @var String */
	public $notes;

	/** @var zauiactivity_dto_UnitItemReserveRequest[] */
	public $unitItems;

}
?>